src/s2geography/geoarrow.h

```diff
@@ -18,10 +18,10 @@ const char* version();
 
 S2::Projection* lnglat();
 
-/// \brief Options used to build Geography objects from GeoArrow arrays
-class ImportOptions {
+/// \brief Options used to import/export Geography objects from/to GeoArrow arrays
+class ImportExportOptions {
```

Comment:
I didn't yet started supporting options in my draft, but was wondering: maybe a separate options class makes more sense? 

There is certainly some overlap (projection / tesselate_tolerance), but several options are also specific (I don't think that oriented and check makes sense for export? And for export to WKT we might want to control the significant digits)